### ISSUE TITLE
offloadcoredata.sh should verify that hardcoded paths exist

### ISSUE THREAD

**Issue by epiccurious** [epiccurious/bitcoin-core-node-builder#6]
Ran into an issue where my `data_directory_target` was pointing to the wrong directory.

The first thing the script should do is check if the source and target paths are valid / exist.

If either the source or target are invalid, `exit 1` the script.

**Comment by epiccurious** [comment]
Also check if the paths end in a trailing slash or not (it's not supposed to) and if so, remove the trailing slash.

Also check if the paths end in two trailing slashes and if they do, exit the program.

**Comment by epiccurious** [comment]
Also, script should verify write access, not read-only, before starting the data transfer.

**Comment by epiccurious** [comment]
`offloadcoredata.sh` issues will not be part of the MVP milestone.

**Comment by epiccurious** [comment]
Closing this issue with the WONTFIX tag for now since the future of `offloadcoredata` is uncertain.
